### PR TITLE
DAO Rupandehi

### PR DESCRIPTION
**Pull request by dashing-dev**
added details of District administration office of Rupandehi

### CODE REVIEW

File: data/websites.json [hunk 1]
```diff
@@ -504,7 +504,13 @@
       "name": "Tilottama Municipality",
       "url": "https://www.tilottamamun.gov.np/en",
       "description": "Official Website Of Tilottama Municipality"
+    },
+    {
+      "name": "District Administration Office of Rupandehi",
+      "url": "https://daorupandehi.moha.gov.np/en",
+      "description": "Official Website Of   District  Administration office of Rupandehi"
```
Comment (sudipsigdel) [inline data/websites.json:511]: "description": "Official Website of District Administration Office of Rupandehi"

**Review by sudipsigdel — CHANGES_REQUESTED** [review]

File: data/websites.json (hunk 1 above)
Comment (sudipsigdel) [inline data/websites.json:511]: "description": "Official Website of District Administration Office of Rupandehi"

**Review by sudipsigdel — APPROVED** [review]
"description": "Official Website of District Administration Office of Rupandehi"

You can use this just to match the formatting.

**Comment by dashing-dev** [comment]
> "description": "Official Website of District Administration Office of Rupandehi"
> 
> You can use this just to match the formatting.
please check

File: data/websites.json [hunk 2]
```diff
@@ -504,7 +504,13 @@
       "name": "Tilottama Municipality",
       "url": "https://www.tilottamamun.gov.np/en",
       "description": "Official Website Of Tilottama Municipality"
+    },
+    {
+      "name": "District Administration Office of Rupandehi",
+      "url": "https://daorupandehi.moha.gov.np/en",
+      "description": "Official Website Of District  Administration Office of Rupandehi"
```
Comment (TheLearneer) [inline data/websites.json:511]:
```suggestion
      "description": "Official Website Of District Administration Office of Rupandehi"
```

**Review by TheLearneer — CHANGES_REQUESTED** [review]
A very small change and we are good to go

**Comment by dashing-dev** [comment]
Done

**Review by TheLearneer — APPROVED** [review]
LGTM

**Pull request merged**